nfs-ganesha/build/setup

```diff
@@ -46,6 +46,9 @@ install_python_packages "pkgs[@]"
 
 NFS_GANESHA_BRANCH=`branch_slash_filter $NFS_GANESHA_BRANCH`
 CEPH_BRANCH=`branch_slash_filter $CEPH_BRANCH`
+BRANCH=${NFS_GANESHA_BRANCH}
+# set flavor as ceph branch libs are coming from
+FLAVOR=${CEPH_BRANCH}
```

Comment:
`$CEPH_BRANCH` defaults to master for when nfs-ganesha's github triggers the build, That's not a bad idea I'm going to do that.